packages/client-python/src/rocketride/_engine/state.py

```
@@ -0,0 +1,231 @@
+
+    def __init__(self):
+        self._db: Optional[aiosqlite.Connection] = None
+
+    async def __aenter__(self):
+        await self.open()
+        return self
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
+        await self.close()
+
+    async def open(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `open`

Add return type annotation: `None`

(ANN202)